src/uri.cr

```diff
@@ -426,4 +426,22 @@ class URI
       URI.escape(password, io)
     end
   end
+
+  @@DEFAULT_PORTS = {
```

Comment:
Use a constant for this (no @@)
```
DEFAULT_PORTS = ...
```

Reply:
Thanks for that. I've changed the class variable to be a constant.